html/elements/link.json

```
@@ -714,6 +714,58 @@
               }
             }
           },
+          "icon": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "4",
+                  "notes": "If both ICO and PNG are available, will use ICO over PNG if ICO has better matching sizes set. (Per caniuse.com.)"
+                },
+                "chrome_android": {
+                  "version_added": "18"
+                },
+                "edge": {
+                  "version_added": "12",
+                  "notes": "In version 79 and later (Blink-based Edge), if both ICO and PNG are available, will use ICO over PNG if ICO has better matching sizes set. (Per caniuse.com.)"
+                },
+                "firefox": {
+                  "version_added": "2"
+                },
+                "firefox_android": {
+                  "version_added": "68"
```

Caniuse doesn't display older versions for some browsers for some reason. Given it was so early in the engine per the Firefox Desktop version, I would say this was also present in the early Firefox Android version.

```suggestion
                  "version_added": "4"
```

---

Thanks for catching this!